dsl_lang/lib/util.ml

```diff
@@ -22,4 +22,20 @@ let parse_file (f : string) =
   let lexbuf = Lexing.from_channel (open_in f) in 
   try parse lexbuf with
     | Parser.Error -> prerr_endline (syntax_error_msg lexbuf); exit 1
-    | Lexer.Err -> prerr_endline (syntax_error_msg lexbuf); exit 1
\ No newline at end of file
+    | Lexer.Err -> prerr_endline (syntax_error_msg lexbuf); exit 1
+
+(** Helper function to turn policy lists into strings. **)
+let rec string_of_plist (plist : Ast.policy list) :string =
+  match plist with
+  | [] -> ""
+  | h :: t -> string_of_policy h ^ ", " ^ string_of_plist t
```

Comment:
A little cleverness here will save you from those trailing commas in the final answers!